lib/Table/TableAutomaticStylesStratum.ts

```diff
@@ -166,11 +160,54 @@ export default class TableAutomaticStylesStratum extends LoadableStratum(
       return true;
   }
 
+  @computed get showDisableTimeOption() {
+    // Return nothing if no row groups or if time column doesn't have at least one interval
+    if (
+      this.catalogItem.activeTableStyle.rowGroups.length === 0 ||
+      !this.catalogItem.activeTableStyle.moreThanOneTimeInterval
+    )
+      return undefined;
+
+    // Return true if at least 50% of rowGroups only have one unique time interval (i.e. they don't change over time)
+    let flat = 0;
+
+    for (
+      let i = 0;
+      i < this.catalogItem.activeTableStyle.rowGroups.length;
+      i++
+    ) {
+      const group = this.catalogItem.activeTableStyle.rowGroups[i];
+      const dates = group[1]
+        .map(
+          row =>
+            this.catalogItem.activeTableStyle.timeColumn?.valuesAsDates.values[
+              row
+            ]
+        )
+        .filter(date => date) as Date[];
+      if (uniq(dates).length <= 1) flat++;
```

Comment:
i don't think uniq works with native array of Dates. https://jsfiddle.net/zm5yd2j7/

Reply:
Yeah nice pickup!
I have changed it to date -> date.getTime() and it now works